_component_design/popovers.md

```diff
@@ -84,22 +88,25 @@ The most defining aspects of a popover are the container and the pointer. The co
 
 <section class="static-section" markdown="1">
 
-## Pattern specifications
-
 <div class="hxRow"  markdown="1">
 
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-### Popover Spacing
+### Style
```

Comment:
redline change: I recommend we refer to the shadow system for the shadow specs rather than detailing the shadow here.